File: C++/Arreglos5.cpp
#include <iostream>
using namespace std;

int main() {
    int arr[3][3] = { {1,2,3}, {4,5,6}, {7,8,9} };

    for(int j = 0; j < 3; j++) {       // Columnas primero
        for(int i = 0; i < 3; i++) {   // Filas después
            cout << arr[i][j] << " ";
        }
        cout << endl;
    }

    return 0;
}

File: C#/Arreglos5.cs
using System;

class Program {
    static void Main() {
        int[,] arr = { {1,2,3}, {4,5,6}, {7,8,9} };

        for(int j = 0; j < 3; j++) {      // Columnas
            for(int i = 0; i < 3; i++) {  // Filas
                Console.Write(arr[i,j] + " ");
            }
            Console.WriteLine();
        }
    }
}


